src/tests/gov/nasa/jpf/test/mc/basic/AttrsTest.java

```diff
@@ -360,35 +360,50 @@ int goModel (double d, int i) {
       System.out.println(attr);
     }
   }
+
+
+
+
+
+  // Constructor based initialization is deprecated in Java 17. So we have to replace with "WrapperClass.valueOf(...)" based initialization.
+  // This below test has been commented out because we have an error when we replace the old constructor type initialization of Wrapper Class of primitive type int with "WrapperClass.valueOf(..)."
+  // Example : { Integer o = new Integer(v) } works but { Integer o = Integer.valueOf(v) } don't.
+
+//  @SuppressWarnings("deprecation")
+//  @Test public void testInteger() {
+//    if (verifyNoPropertyViolation()) {
+//      int v = 42;
+//      Verify.setLocalAttribute("v", 4200);
+//
+//      // explicit
+//      Integer o = Integer.valueOf(v);
+//      int j = o.intValue();
+//      int attr = Verify.getLocalAttribute("j");
+//      assert attr == 4200;
+//
+//      // semi autoboxed
```

Comment:
If the part above works, we can start comment out this part.
I think it is nicer to use `/*` here, akin to the commented-out part below. Definitely not `//` in addition to `/*`.
With `/*`...`/*`, we clearly mark two blocks of code as being parts of the test that do not (or do no longer) work. With `//`, we lose that information.

Reply:
Actually, none of the Assertion works.

Pavel taught me that `Local Variable` and `Local Attribute` are different entities in JPF.
Constructor type initialisation passes the value of `Local Variables` to `Local Attributes` but in case of new method it doesn't.